man/systemd.netdev.xml

```diff
@@ -399,8 +399,14 @@
             This option is compulsory.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>GVRP=</varname></term>
+          <listitem>
+            <para>The Generic VLAN Registration Protocol (GVRP) is a protocol that
+            allows automatic learning of VLANs on a network. A boolean. Defaults to unset.</para>
```

Comment:
Maybe it should be changed to "Defaults to system settings"? 
true=enabled
false=disabled
unset=System Default?